Sources/LiveViewNative/Views/Layout Containers/Collection Containers/Section.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Section.swift
+//  
+//
+//  Created by Carson Katri on 2/2/23.
+//
+
+import SwiftUI
+
+struct Section<R: CustomRegistry>: View {
+    @ObservedElement private var element: ElementNode
+    private let context: LiveContext<R>
+    
+    init(element: ElementNode, context: LiveContext<R>) {
+        self.context = context
+    }
+    
+    public var body: some View {
+        #if os(macOS)
+        self.section
+            .collapsible(isCollapsible)
+        #else
+        self.section
+        #endif
+    }
+    
+    private var isCollapsible: Bool {
+        guard let collapsible = element.attribute(named: "collapsible")
+        else { return false }
+        guard let value = collapsible.value
+        else { return true }
+        return Bool(value) ?? false
+    }
```

Comment:
This allows the following:
```html
<section collapsible>
  ...
</section>
```

@shadowfacts Is there already something to get an attribute as a boolean, and if it has no value treat it as true? If not, I can extract this out because it seems like it would be useful for other boolean attributes.

Reply:
There is not currently, but I agree that would be useful to extract out.

Reply:
Interestingly, if we treat these as [boolean attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#boolean_attributes) the spec states:

> The values "true" and "false" are not allowed on boolean attributes. To represent a false value, the attribute has to be omitted altogether.

LV automatically omits the attribute if `false` is passed, and passes a `nil` value if `true` is passed: